openmrs-19/src/main/resources/webapp/messages/messages_es.properties

```diff
@@ -0,0 +1,20 @@
+#Module name
```

Comment:
I thought you only needed to add these files if they were different, can you confirm?